projects/wave-core-new/src/lib/layer-collections/layer-collection-dropdown/layer-collection-dropdown.component.html

```diff
@@ -17,16 +17,17 @@
             </mat-select>
         </mat-form-field>
 
-        <div *ngFor="let prop of properties[i]">
-            <p>
-                <em
-                    ><span class="label">{{ prop[0] }}</span> {{ prop[1] }}</em
-                >
-            </p>
-        </div>
-
-        <div *ngIf="descriptions[i].length > 0">
-            <p class="mat-caption label justify">{{ descriptions[i] }}</p>
+        <div *ngIf="collections[i + 1] as selected">
+            <div *ngFor="let prop of selected.properties">
+                <p>
+                    <em
+                        ><span class="label">{{ prop[0] }}</span> {{ prop[1] }}</em
```

Comment:
Maybe you can use this?
https://angular.io/api/common/KeyValuePipe

Reply:
it's not a map but an array

Reply:
But look at the example, you could put this into the constructor of a Map, maybe.

Reply:
but how would that be better? It's already so simple.